packages/terra-list/src/MultiSelectList.jsx

```diff
@@ -111,15 +111,16 @@ class MultiSelectList extends React.Component {
   cloneChildItems(items) {
     const disableUnselectedItems = this.state.selectedIndexes.length >= this.validatedMaxCount();
 
-    return items.map((item, index) => {
-      const wrappedOnClick = this.wrappedOnClickForItem(item, index);
-      const wrappedOnKeyDown = this.wrappedOnKeyDownForItem(item, index);
-      const newProps = this.newPropsForItem(item, index, wrappedOnClick, wrappedOnKeyDown, disableUnselectedItems);
```

Comment:
I have no idea what you are talking about. If I did remove disableUnselectedItems, I would have amended that commit and no one would be the wiser :)